src/threading/Manager.cc

```diff
@@ -22,6 +22,8 @@ void HeartbeatTimer::Dispatch(double t, bool is_expire) {
 
 } // namespace detail
 
+std::vector<uint64_t> pending_bucket_brackets = {1, 10, 100, 1000, 10000, std::numeric_limits<uint64_t>::infinity()};
```

Comment:
Should this be `static` or in an anonymous namespace if we have one?

The zeekier way would probably to have this in BifConst as a vector of count (so it could also be disabled if its empty), though not sure it's worth it really, but it might make testing easier.

I'd also randomly add 100000, but that's just a hunch :-)